### ISSUE TITLE
Add DB format v4 support

### ISSUE THREAD

**Issue by rgl** [tkuhlman/gopwsafe#1]
It would be nice to have support for the [DB format v4](https://github.com/pwsafe/pwsafe/blob/master/docs/formatV4.txt), at least because of its new attachments types (e.g. images).

I would drop the support for writing in the v3 format and automatically convert to v4.

Also, can we have the DB handling code in its own repository (e.g. pwsafe)? I guess that would make it easier to be used by other applications.

**Comment by tkuhlman** [comment]
I agree DB format v4 is a great feature to add and something I would like to see but am not working on myself right now. I will take a contribution or perhaps someday get to work on it myself.

I don't think the db code is hard to use from other applications right now. Just "go get "github.com/tkuhlman/gopwsafe" then in your code "import github.com/tkuhlman/gopwsafe/pwsafe". If there are suddenly other apps using my code that need this or other contributors who have this need I'd be willing to change but as of now it works well for me to have it all in a single repo.
